src/process_controller.cpp

```diff
@@ -219,6 +218,13 @@ void ProcessController::handle_ptrace_event(pid_t child, int event)
         }
     }
     break;
+    case PTRACE_EVENT_EXEC:
+    {
+        std::string name = get_process_comm(child);
```

Comment:
Once the process execs, all other threads of the process are gone, so do we need some cleanup action here?

Reply:
```
If the PTRACE_O_TRACEEXIT option is on, PTRACE_EVENT_EXIT will happen before actual  death.   This  applies to [...], and when threads are torn down on execve(2) in a multithreaded process.
```

What we have no should suffice